python_modules/dagster/dagster/_core/definitions/time_window_partitions.py

```diff
@@ -106,14 +126,15 @@ def __new__(
         cls,
         start: Union[datetime, str],
         fmt: str,
-        end: Union[datetime, str, None] = None,
-        schedule_type: Optional[ScheduleType] = None,
         timezone: Optional[str] = None,
+        end: Union[datetime, str, None] = None,
```

Comment:
Is it okay to restructure the order of args here? I forget if we require keyword args

Reply:
Alas we never put a `*` in the arg list to require keyword args, so technically we could break someone by reordering them.